contrail-agent/hooks/docker_utils.py

```diff
@@ -16,13 +20,25 @@
 DOCKER_COMPOSE_CLI = "docker-compose"
 
 
+def _format_curl_https_proxy_opt():
+    proxy_settings = env_proxy_settings(['https'])
+    https_proxy = None
+    if proxy_settings:
+        https_proxy = proxy_settings.get('https_proxy')
+        return '--proxy {}'.format(https_proxy) if https_proxy else ''
+    return ''
+
+
 def install():
     apt_install(["apt-transport-https", "ca-certificates", "curl",
                  "software-properties-common"])
-    cmd = ["/bin/bash", "-c",
-           "set -o pipefail ; curl -fsSL --connect-timeout 10 "
-           "https://download.docker.com/linux/ubuntu/gpg "
-           "| sudo apt-key add -"]
+    cmd = [
+        "/bin/bash", "-c",
+        "set -o pipefail ; curl {}"
```

Comment:
please add space after {} in all 4 files.

Reply:
Thanks for catching!